sklearn/cluster/affinity_propagation_.py

```diff
@@ -197,7 +197,10 @@ class AffinityPropagation(BaseEstimator, ClusterMixin):
     Parameters
     ----------
     damping : float, optional, default: 0.5
-        Damping factor between 0.5 and 1.
+        Damping factor (between 0.5 and 1) damps the
```

Comment:
Yes, that explains damping better. Should I add that in both API doc and User guide?